lib/LaTeXML/Post/Graphics.pm

```diff
@@ -98,6 +98,21 @@ sub process {
   $doc->closeCache;    # If opened.
   return $doc; }
 
+# Need a proper API to query PI's, probably in Post
+# But Core isn't respecting @at@document@begin from preloaded packages,
+# so here we're testing BOTH a straight DPI PI and the options from latexml.sty (ugh)
+sub getDPI {
+  my ($self, $doc) = @_;
+  my $dpi;
+  foreach my $pi (@{ $$doc{processingInstructions} }) {
+    if ($pi =~ /^\s*DPI\s*=\s*([\"\'])(.*?)\2\s*$/) {
```

Comment:
wrote a small script to test this, and it looks as if this `\2` should really be `\1` here.

It's still `\2` in the package regex, which is what ends up matching.

Reply:
Good catch! It should be 1. The `elsif` is an ugly fallback, only needed because `--preload` doesn't respect `@at@document@begin`, which would have added a proper PI.